openmdao/docs/features/building_blocks/components/exec_comp.rst

```diff
@@ -179,5 +179,14 @@ that `fd` should be used for a given derivative is to call :code:`declare_partia
     openmdao.components.tests.test_exec_comp.TestFunctionRegistration.featuretest_register_simple_unsafe
     :layout: interleave
 
+ExecComp Example: Adding Expressions
+------------------------------------
+
+You can add additional expressions to an `ExecComp` with the "add_expr" method.
```

Comment:
I wonder if the example of using `add_expr` in `configure` might be better for the docs, as it gives a bit of rationale for wanting to add it later...

Reply:
Yeah, that's more likely. I'll swap the example.